src/SDKs/ContainerInstance/ContainerInstance.Tests/ContainerInstance.Tests.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ContainerInstance.Tests</PackageId>
+    <AssemblyName>ContainerInstance.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Utilities\ContainerInstanceTestUtilities.cs" />
```

Comment:
@ScottHolden this is not needed. I know VS might have added it, can you remove it. VS will include code files by default in all sub directories.